deploy/cloud/operator/docs/crd-ref-docs-config.yaml

```
@@ -0,0 +1,45 @@
+    - "metadata.uid"
+    - "status.conditions[*].lastTransitionTime"
+    - "status.observedGeneration"
+    - "TypeMeta$"
+  ignoreTypes:
+    - "List$"
+    - "ParseError$"
+
+render:
+  # Output format - use markdown instead of default asciidoc
+  format: markdown
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)